### PR TITLE
Make the license header consistent.

### PR DESCRIPTION
**Pull request by DivvSaxena**
Fixes #ISSUSE_ID.

Changes proposed in this pull request:
  -Making the license header consistent.
  - Issue #21643
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: src/resources/spotless_shardingsphere_formatter.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License 
+ * , Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.
+ * You may obtain a copy of the License at
```
Comment (RaigorJiang) [inline src/resources/spotless_shardingsphere_formatter.xml:10]: In fact, we should not use `*` as a comment symbol in XML files.
Refer to https://github.com/apache/shardingsphere/blob/master/pom.xml

Comment (DivvSaxena) [reply src/resources/spotless_shardingsphere_formatter.xml]: @RaigorJiang  Updated it (commit) .

**Review by RaigorJiang — DISMISSED** [review]

**Comment by RaigorJiang** [comment]
@DivvSaxena 
Sorry, the license check failed, let's investigate.

File: .gitattributes [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License
+# , Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License.
+# You may obtain a copy of the License at
 #
 #   http://www.apache.org/licenses/LICENSE-2.0
 #
```
Comment (RaigorJiang) [inline .gitattributes:11]: Is it because you only modified one part of the license and not the second half?

Comment (DivvSaxena) [reply .gitattributes:11]: I am not getting it , I made the license header uniform and in all files i have used the same format , so how to fix this problem ?

Comment (DivvSaxena) [reply .gitattributes:10]: Check it now i fixed that problem

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**